site/content/docs/mdx.md

```diff
@@ -299,3 +299,119 @@ You can also use comments in expressions with JavaScript's multiline comment syn
 ```md
 {/* A comment! */}
 ```
+
+## MDX components
```

Comment:
@olayway the one thing i'm not clear about reading this is **why** i would use mdx vs jsx extension for my components. i.e. when would i use "MDX" components vs standard components? is it when i want to use markdown? Would it be worth adding a very brief explanation of *why* you would use mdx components vs standard jsx ones?

Reply:
You're right, I'll add a few sentences :)